Assets/MRTK/SDK/Features/UX/Scripts/Pointers/PokePointer.cs

```diff
@@ -213,7 +215,6 @@ private bool FindClosestTouchableForLayerMask(LayerMask layerMask, out BaseNearI
             {
                 closest = null;
                 closestDistance = float.PositiveInfinity;
-                closestNormal = Vector3.zero;
```

Comment:
Hi @jonathoncobb, the removal of this line introduces a compilation error as the complier expects `closestNormal` to be assigned given it is an out parameter. Could you add it back or assign it a different default value (maybe Vector3.forward)?

Reply:
done